lms/envs/common.py

```diff
@@ -5379,6 +5382,26 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 NOTIFICATIONS_EXPIRY = 60
 EXPIRED_NOTIFICATIONS_DELETE_BATCH_SIZE = 10000
 
+######################## Event bus producer config ########################
+
+# .. setting_name: EVENT_BUS_PRODUCER_CONFIG
+# .. setting_default: {}
+# .. setting_description: Dictionary of event_types mapped to lists of dictionaries containing topic related configuration.
+#    Each topic configuration dictionary contains
+#    * a topic/stream name called `topic` where the event will be pushed to.
```

Comment:
Nit: I think `* topic: a topic/stream ...` would be a format that is quicker to read.